molo/profiles/templates/profiles/viewprofile.html

```diff
@@ -23,14 +22,41 @@ <h1 class="heading heading--profiles">{% trans "Hello " %}{{ user.username }}</h
     </fieldset>
 
     {% if settings.profiles.UserProfilesSettings.show_mobile_number_field and settings.profiles.UserProfilesSettings.country_code %}
+        <fieldset>
+       <label for="id_mobile_number">{% trans "Mobile number:" %}</label>
+       <p class="profile__mobile-number">{{ user.profile.mobile_number | default:"Not set" }}</p>
+       {{ form.mobile_number.errors }}
+       {{ form.mobile_number }}
```

Comment:
@Ntu2koTrevor on the view profile page we only need to show the field data here, so we don't need to show the input boxes here, please remove them for all the filed